### PR TITLE
8283217: Leak FcObjectSet in getFontConfigLocations() in fontpath.c

### PR DESCRIPTION
**Pull request by zhengyu123**
Please review this small patch to fix leaking of `FcObjectSet` in  `getFontConfigLocations()`

Test:
- [x]  jdk_2d

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8283217](https://bugs.openjdk.java.net/browse/JDK-8283217): Leak FcObjectSet in getFontConfigLocations() in fontpath.c

### Reviewers
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**)
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/7826/head:pull/7826` \
`$ git checkout pull/7826`

Update a local copy of the PR: \
`$ git checkout pull/7826` \
`$ git pull https://git.openjdk.java.net/jdk pull/7826/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 7826`

View PR using the GUI difftool: \
`$ git pr show -t 7826`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/7826.diff">https://git.openjdk.java.net/jdk/pull/7826.diff</a>

</details>

### CODE REVIEW

**Review by prrace — APPROVED** [review]

**Comment by zhengyu123** [comment]
Friendly ping! May I have a second review.

Thanks,

**Review by aivanov-jdk — APPROVED** [review]

**Comment by zhengyu123** [comment]
Thanks, @prrace @aivanov-jdk

**Pull request closed without merging**
